src/build.py

```diff
@@ -805,6 +809,7 @@ def LLVM():
   build_dylib = 'ON' if not IsWindows() else 'OFF'
   command = CMakeCommandNative([
       GetLLVMSrcDir('llvm'),
+      '-DCMAKE_CXX_FLAGS=-Wno-nonportable-include-path',
```

Comment:
Why this?

Reply:
There are a few places where the filename on disk differs in case from the filename in the #include statement, which produces a warning. There aren't a lot but they are included indirectly in basically every file, and it results in a lot of log spam.

Reply:
Actually I saw this in Binaryen too and looked again. It looks like in that case the name actually does match (e.g. both the [filename](https://github.com/WebAssembly/binaryen/blob/master/src/cfg/Relooper.h) in the repo and the include start with a cap; but the [warning](https://logs.chromium.org/logs/emscripten-releases/buildbucket/cr-buildbucket.appspot.com/8897430022390414848/+/steps/Build_Binaryen/0/stdout) complains that the path on disk has no cap. So maybe there's something about how the repo is cloned or checked out.

Reply:
Was the file renamed at some point?   Perhaps a clobber build?